modeltranslation/translator.py

```diff
@@ -124,6 +125,43 @@ def __str__(self):
         return '%s: %s + %s' % (self.__class__.__name__, local, inherited)
 
 
+class MultilingualOptions(options.Options):
+    """ Far from a MetaClass expert, maybe this could mbe combined with TranslationOptions"""
+
+    @cached_property
+    def base_manager(self):
```

Comment:
Can we not copy-paste django code, but instead just call `super()`, check if Manager is not Multilingual and override __class__?

Reply:
I found it a bit dirty myself as well, I think meanwhile i figured out a way that might work. Coming back to it